scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2915,7 +2915,7 @@ u64 antistall_set(u64 dsq_id, u64 jiffies_now)
 		if (!(p = bpf_task_from_pid(__p->pid)))
 			continue;
 
-		if (!(taskc = lookup_task_ctx(p)))
+		if (!(taskc = lookup_task_ctx_may_fail(p)))
```

Comment:
Strongly oppose this one. It's better we get a failure due to task_ctx failure here than a stall which could happen due to a myriad of reasons. Maybe wait to see this be a problem before making this change.